Tools/build.bat

```diff
@@ -76,8 +76,8 @@ for %%N in (%SUPPORTED_BUILD_SPECS%) do (
   )
 )
 
-rem Silently exit if the build target could not be matched
-goto :eof
+rem Return error if the build target could not be matched
+goto :fail
```

Comment:
Cool! I will update the PR. I would also recommend adding a comment there to make it clear.